### PR TITLE
hack for MAX31865

### PR DESCRIPTION
**Pull request by clarsen**
also, somewhat not appropriate to be merged yet, but for your review.

### CODE REVIEW

**Comment by clarsen** [comment]
Updated with documentation and ability to pass in GPIO settings...

File: SR700_Artisan_Server/Start_SR700_Artisan_Server.py [hunk 1]
```diff
@@ -129,6 +148,7 @@ def output_current_state(self):
         return ret_state
 
     def output_sr700_and_phidget_temp(self):
+        raise Exception("Shouldn't be called")
```
Comment (lucapinello) [inline SR700_Artisan_Server/Start_SR700_Artisan_Server.py]: Why do we have this raise?

File: SR700_Artisan_Server/Roaster_Cooling.py [hunk 2]
```diff
@@ -12,6 +12,14 @@ def main():
     roast_control.enable_fan_manual_mode()
     roast_control.set_fan_speed(9)
     roast_control.run_cooling()
+
+    # if the Artisan alarms trigger before detecting manual fan mode, then 
+    # they will reset the fan to a (likely) lower speed.  So, keep 
+    # keep setting fan to recover from race condition.
+    time.sleep(2)
+    roast_control.set_fan_speed(9)
+    time.sleep(2)
```
Comment (lucapinello) [inline SR700_Artisan_Server/Roaster_Cooling.py]: very interesting, I noticed before something was not correct. This should fix it

Comment (clarsen) [reply SR700_Artisan_Server/Roaster_Cooling.py]: Yeah I further had to add a bit more delay to trigger the fan over the course of 10 seconds or so.

**Pull request merged**